index.bs

```diff
@@ -1346,6 +1346,31 @@ Removing AB or TAG members</h5>
 
 	Note: Removal from the [=AB=] or [=TAG=] does not imply removal from a convened [=Council=].
 
+<h5 id="AB-TAG-recall">
+AC Recall of AB or TAG</h5>
+
+	The [=Advisory Committee=] <em class=rfc2119>may</em>
+	<dfn export>recall</dfn>,
+	as a whole,
+	all [[#AB-TAG-elections|elected]] and [[#TAG-appointments|appointed]] participants
+	from the [=Advisory Board=] or the [=Technical Architecture Group=].
+
+	An [=Advisory Committee representative=] invokes a [=recall=]
```

Comment:
It seems like the bar for triggering this should be much higher than a single AC representative request.

Reply:
read on. This starts with a single person (how else), but then you need to meet the same criteria as an AC appeal, which means finding 5% needs to support having the vote, or it stops right there.